src/components/Board.js

```diff
@@ -9,12 +9,16 @@ const generateSquareComponents = (squares, onClickCallback) => {
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
+  const oneDarray = [].concat(...squares)
+  return oneDarray.map((square) => {
+      return <Square value={square.value} id={square.id} onClickCallback={onClickCallback} />
```

Comment:
You're missing the `key` prop

```suggestion
      return <Square key={square.id} value={square.value} id={square.id} onClickCallback={onClickCallback} />
```